hack/devtools/msi.sh

```diff
@@ -13,11 +26,15 @@ if [[ -z "$sp" ]]; then
     echo "Failed to create mock MSI object"
     exit 1
 fi
-clientID=$(get_mock_msi_clientID "$sp")
-tenantID=$(get_mock_msi_tenantID "$sp")
+mockClientID=$(get_mock_msi_clientID "$sp")
+mockTenantID=$(get_mock_msi_tenantID "$sp")
 base64EncodedCert=$(get_mock_msi_cert "$sp")
 
+setup_platform_identity
+cluster_msi_role_assignment "${mockClientID}"
+
 # Print the extracted values
-echo "Client ID: $clientID"
-echo "Tenant ID: $tenantID"
-echo "Base64 Encoded Certificate: $base64EncodedCert"
\ No newline at end of file
+echo "Cluster MSI Client ID: $mockClientID"
+echo "Cluster MSI Tenant ID: $mockTenantID"
+echo "Cluster MSI Base64 Encoded Certificate: $base64EncodedCert"
```

Comment:
nit: This isn't a change you introduced, but if we're here anyways... If I'm reading all this correctly - this is a private key? It's best to not print secrets to STDOUT even if it's obfuscated by base64 (encoding != encrypting).